org.getmonero.monero.desktop.xml

```diff
@@ -0,0 +1,64 @@
+
```

Comment:
You're not installing this file. It needs to be defined in `sources` array and installed in `build-commands`.

Reply:
also its name should be `org.getmonero.Monero.metainfo.xml` (note the case)